src/Search/SearchEngine.php

```diff
@@ -128,6 +128,17 @@ public static function getMetaItemtypeAvailable($itemtype): array
                 continue;
             }
 
+            if ($key === 'itil_types') {
+                $timeline_types = [\ITILFollowup::class, \CommonITILTask::class, \ITILSolution::class, \CommonITILValidation::class];
+                foreach ($timeline_types as $timeline_type) {
+                    if (is_a($item, \CommonITILTask::class) || is_a($item, \CommonITILValidation::class)) {
+                        $linked[] = $item->getItilObjectItemType();
+                    } else if (is_a($item, $timeline_type)) {
+                        $linked = [...$linked, ...$values];
+                    }
+                }
+            }
```

Comment:
```suggestion
            if ($key === 'itil_types') {
                if (is_a($item, \CommonITILTask::class) || is_a($item, \CommonITILValidation::class)) {
                    $linked[] = $item->getItilObjectItemType();
                } else {
                    $timeline_types = [\ITILFollowup::class, \ITILSolution::class];
                    foreach ($timeline_types as $timeline_type) {
                        if (is_a($item, $timeline_type)) {
                            $linked = [...$linked, ...$values];
                        }
                    }
                }
            }
```

In case of Task and Validation, final array was `['Ticket', 'Ticket', 'Ticket']` when testing it in webhooks (Ticket task - new)